### PR TITLE
chore: upgrade to csi-proxy v0.2.0 beta

### PR DESCRIPTION
**Pull request by andyzhangx**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
chore: upgrade to csi-proxy v0.2.0 beta

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

```console
TZ=UTC 
git --no-pager show \
  --quiet \
  --abbrev=12 \
  --date='format-local:%Y%m%d%H%M%S' \
  --format="%cd-%h"
```

**Release note**:
```
none
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -236,7 +236,7 @@ replace (
 	github.com/kr/pretty => github.com/kr/pretty v0.1.0
 	github.com/kr/pty => github.com/kr/pty v1.1.5
 	github.com/kr/text => github.com/kr/text v0.1.0
-	github.com/kubernetes-csi/csi-proxy/client => github.com/kubernetes-csi/csi-proxy/client v0.0.0-20200330215040-9eff16441b2a
+	github.com/kubernetes-csi/csi-proxy/client => github.com/kubernetes-csi/csi-proxy/client v0.0.0-20200907051941-1c32ab57cb4f
```
Comment (jingxu97) [inline go.mod:239]: this replace is needed?

Comment (andyzhangx) [reply go.mod:239]: yes, while in this PR, there is no vendor code change since smb client is still alpha and remains the same.

**Pull request merged**